src/Raven.Client/Documents/Indexes/IndexDefinitionBuilder.cs

```diff
@@ -167,7 +167,17 @@ public IndexDefinition ToIndexDefinition(DocumentConventions conventions, bool v
                 var spatialOptions = ConvertToStringDictionary(SpatialIndexes);
 
                 if (conventions.PrettifyGeneratedLinqExpressions)
+                {
                     indexDefinition.Reduce = IndexPrettyPrinter.TryFormat(indexDefinition.Reduce);
+                    var maps = indexDefinition.Maps.Select(x => IndexPrettyPrinter.TryFormat(x));
```

Comment:
I do not think that `indexDefinition.Maps` are filled at this point. This is done 50 lines below.